src/pages/[locale]/resources.tsx

```diff
@@ -114,112 +125,155 @@ const ResourcesPage = ({ txCostsMedianUsd }) => {
+                          {items.map((item) => (
+                            <ResourceItem item={item} key={item.title} />
+                          ))}
+                        </ResourcesContainer>
+                      </div>
+                    </div>
+                  ))}
                 </div>
-              </div>
-            ))}
-          </div>
-        </section>
-      ))}
-
-      <div className="mb-20 text-center">
-        <Translation id="page-resources:page-resources-find-more" />
-      </div>
-
-      <Section
-        id="contribute"
-        className={cn(
-          "before:absolute before:-inset-px before:bottom-0 before:z-hide before:rounded-[calc(theme(borderRadius.4xl)+1px)] before:content-['']", // Border/gradient positioning
-          "before:bg-gradient-to-b before:from-primary-hover/[0.24] before:to-primary-hover/[0.08] before:dark:from-primary-hover/40 before:dark:to-primary-hover/20", // Border/gradient coloring
-          "relative inset-0 rounded-4xl bg-background" // Paint background color over card portion
-        )}
-      >
-        <div className="mb-12 flex flex-col gap-y-8 rounded-4xl bg-radial-a px-8 py-12 lg:mb-32 xl:mb-36">
-          <div className="flex flex-col gap-y-4 text-center">
-            <h2>{t("page-resources-contribute-title")}</h2>
-            <p>{t("page-resources-contribute-description")}</p>
-          </div>
-          <div className="mx-auto grid grid-cols-1 gap-16 md:grid-cols-2">
-            {/* TODO: Add issue template for resource listing and redirect to new template */}
-            <ButtonLink
-              href={new URL(
-                "issues/new?template=feature_request.yaml",
-                GITHUB_REPO_URL
-              ).toString()}
-              variant="outline"
-              isSecondary
-            >
-              {t("page-resources-suggest-resource")}
-            </ButtonLink>
-            <ButtonLink
-              href={new URL(
-                "issues/new?template=bug_report.yaml",
-                GITHUB_REPO_URL
-              ).toString()}
-              variant="outline"
-              isSecondary
-            >
-              <FaGithub /> {t("page-resources-found-bug")}
-            </ButtonLink>
+              </section>
+            </Stack>
+          ))}
+        </Stack>
+
+        <VStack className="gap-4 py-16">
+          <div className="text-center font-bold">
+            <Translation id="page-resources:page-resources-find-more" />
           </div>
-        </div>
-      </Section>
+          <ButtonLink
+            href="https://ethereumdashboards.com"
+            size="lg"
+            onClick={() => {
+              trackCustomEvent({
+                eventCategory: EVENT_CATEGORY,
+                eventAction: "links",
+                eventName: "ethereumdashboards.com",
+              })
+            }}
+          >
+            ethereumdashboards.com
+          </ButtonLink>
+        </VStack>
```

Comment:
This get's a little complicated with translations, since not languages structures sentences the same as english. That was why the link was nested in the json string, so then translators can the move the link where it fits best within the sentence.

@lukassim Any thoughts if we could get away with this type of trailing button label in regards to translations?

<img width="558" alt="image" src="https://github.com/user-attachments/assets/df65fb07-9927-4977-8866-1375406eeb48" />

Reply:
Yeah good point ideally no links are in a fixed position in regards to the text since sentence structures can be different, this would likely cause issues in some languages

Reply:
@wackerow @lukassim

If ya don't mind 😅 what about keeping the text exclusive from the link?

=====
"Find more great resources!"

<Button />
=====

So, not limit the flexibility of the design, while still being clear of the action. When we have other links that take action on the page that look like buttons, if we kept it as a link here, it might look like there is a broken style.

There is nothing else near this collection in the whitespace, so the clarity should still be maintained.